
Hm - let's use the normal types that are defined (too lazy to serch for these
variants ...).
                                                                                  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.8.1/include/linux/crc-ccitt.h.orig	2004-09-15 19:53:29.752995143 +0200
+++ linux-2.6.8.1/include/linux/crc-ccitt.h	2004-09-15 19:54:02.301801850 +0200
@@ -3,11 +3,11 @@
 
 #include <linux/types.h>
 
-extern u16 const crc_ccitt_table[256];
+extern __u16 const crc_ccitt_table[256];
 
-extern u16 crc_ccitt(u16 crc, const u8 *buffer, size_t len);
+extern __u16 crc_ccitt(__u16 crc, const __u8 *buffer, size_t len);
 
-static inline u16 crc_ccitt_byte(u16 crc, const u8 c)
+static inline __u16 crc_ccitt_byte(__u16 crc, const __u8 c)
 {
 	return (crc >> 8) ^ crc_ccitt_table[(crc ^ c) & 0xff];
 }
